android/app/src/main/java/org/uwaterloo/subletr/api/apis/UserApi.kt

```diff
@@ -52,6 +54,149 @@ class UserApi(basePath: kotlin.String = defaultBasePath, client: OkHttpClient =
         }
     }
 
+    /**
+     * 
+     * 
+     * @param userId 
+     * @return kotlin.String
+     * @throws IllegalStateException If the request is not correctly configured
+     * @throws IOException Rethrows the OkHttp execute method exception
+     * @throws UnsupportedOperationException If the API returns an informational or redirection response
+     * @throws ClientException If the API returns a client error response
+     * @throws ServerException If the API returns a server error response
+     */
+    @Suppress("UNCHECKED_CAST")
+    @Throws(IllegalStateException::class, IOException::class, UnsupportedOperationException::class, ClientException::class, ServerException::class)
+    suspend fun userAvatarGet(userId: kotlin.Int) : kotlin.String = withContext(Dispatchers.IO) {
```

Comment:
just had a question about this, how would you get your own userid so the userAvatarGet call can be made from the my account page. Not sure how I can get the the userid

Reply:
The user_id is encoded into the JWT auth token. The user_id is the payload sub field.